mlrun/feature_store/steps.py

```diff
@@ -6,6 +6,7 @@
 import pandas as pd
 from storey import MapClass
 
+from mlrun.errors import MLRunInvalidArgumentError
```

Comment:
We are trying to use less `from x import y` but rather use `import x`, this reduces the possibility for circular imports as well as makes it better to understand to where certain classes are part of.
```suggestion
import mlrun.errors
```